wakuv2/nwaku.go

```diff
@@ -361,19 +369,23 @@ type WakuPubsubTopic = string
 type WakuContentTopic = string
 
 type WakuConfig struct {
-	Host                 string   `json:"host,omitempty"`
-	Port                 int      `json:"port,omitempty"`
-	NodeKey              string   `json:"key,omitempty"`
-	EnableRelay          bool     `json:"relay"`
-	LogLevel             string   `json:"logLevel"`
-	DnsDiscovery         bool     `json:"dnsDiscovery,omitempty"`
-	DnsDiscoveryUrl      string   `json:"dnsDiscoveryUrl,omitempty"`
-	MaxMessageSize       string   `json:"maxMessageSize,omitempty"`
+	Host        		 string `json:"host,omitempty"`
+	Port        		 int    `json:"port,omitempty"`
+	NodeKey     		 string `json:"key,omitempty"`
+	EnableRelay  		 bool   `json:"relay"`
+	LogLevel     		 string `json:"logLevel"`
+	DnsDiscovery 		 bool `json:"dnsDiscovery,omitempty"`
+	DnsDiscoveryUrl 	 string `json:"dnsDiscoveryUrl,omitempty"`
+	MaxMessageSize		 string `json:"maxMessageSize,omitempty"`
```

Comment:
Not sure why this happens, in my IDE looks aligned, is there a linting tool I should use?